File: solutions/uri/1006/1006.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8')

const [a, b, c] = input
  .trim()
  .split('\n')
  .map(x => parseFloat(x))

console.log(`MEDIA = ${((a * 2.0 + b * 3.0 + c * 5.0) / 10.0).toFixed(1)}`)

File: solutions/uri/1006/1006.r
input <- file('stdin', 'r')

a <- as.double(readLines(input, n=1))
b <- as.double(readLines(input, n=1))
c <- as.double(readLines(input, n=1))

result = sprintf("MEDIA = %.1f", (a * 2.0 + b * 3.0 + c * 5.0) / 10.0)
write(result, "")
